backend/config/config.go

```diff
@@ -601,12 +807,18 @@ func (p *ThirdPartyProvider) Validate() error {
 }
 
 type ThirdPartyProviders struct {
-	Google    ThirdPartyProvider `yaml:"google" json:"google,omitempty" koanf:"google"`
-	GitHub    ThirdPartyProvider `yaml:"github" json:"github,omitempty" koanf:"github"`
-	Apple     ThirdPartyProvider `yaml:"apple" json:"apple,omitempty" koanf:"apple"`
-	Discord   ThirdPartyProvider `yaml:"discord" json:"discord,omitempty" koanf:"discord"`
+	// `apple` contains the provider configuration for Sign in with Apple.
+	Apple ThirdPartyProvider `yaml:"apple" json:"apple,omitempty" koanf:"apple"`
+	// `discord` contains the provider configuration for Discord.
+	Discord ThirdPartyProvider `yaml:"discord" json:"discord,omitempty" koanf:"discord"`
+	// `github` contains the provider configuration for GitHub.
+	GitHub ThirdPartyProvider `yaml:"github" json:"github,omitempty" koanf:"github"`
+	// `google` contains the provider configuration for Google.
+	Google ThirdPartyProvider `yaml:"google" json:"google,omitempty" koanf:"google"`
+	// `linkedin` contains the provider configuration for LinkedIn.
+	LinkedIn ThirdPartyProvider `yaml:"linkedin" json:"linkedin,omitempty" koanf:"linkedin"`
+	// `microsoft` contains the provider configuration for Microsoft.
```

Comment:
The comments should be consistent: `... configuration for Sign in with Apple` - `... configuration for Discord.`